README.md

```diff
@@ -106,10 +135,56 @@ All commands are issued from the the root of the project folder.
  * (e.g. `<project_root>/platforms/ios/ThaliTest.xcodeproj`)
  * `~/Code/ThaliTest> open platforms/ios/ThaliTest.xcodeproj`
 
+### Unit Testing the Thali Cordova Plugin
+
+The Thali Cordova plugin uses the [Jasmine](http://jasmine.github.io/) Behavior Driven Development framework for JavaScript.
+
+#### Desktop Testing
+
+Testing is available on the desktop for the Thali Cordova Plugin which uses a mock object to simulate the native Cordova `Mobile` calls.
+
+To get started, install, Jasmine:
+```
+$ sudo [npm] install -g jasmine
+```
+
+Next, simply run Jasmine which then looks at the specifications in the `specs` folder.  
+```
+$ jasmine
+```
+
+#### Mobile Testing
+
+Testing is also available on the mobile devices as well.  This will use the regular `Mobile` calls directly through Cordova to talk to the underlying system.
+
+To get started, copy the files from `sample/android/www` or `sample/ios/www` to the `www` folder.
+
+Next, install the requirements for testing:
+```
+$ jx install
+```
+
+Finally, build the application and run it on your device.  The test results should be shown in your developer console.
+
+### Documentation
+
+The following API documentation is available for the Thali Cordova Plugin:
+- [Thali Cordova Connectivity API](doc/api/replication.md)
+- [`ThaliReplicationManager` class](doc/api/thalireplicationmanager.md)
+- [`ThaliEmitter` internal class](doc/api/thaliemitter.md)
+
 ### Contributing
+
 If you see a mistake, find a bug, or you think there is a better way to do something, feel free to contribute.
 Email [thali-talk@thaliproject.org](mailto:thali-talk@thaliproject.org) to connect with other contributors and
 get started with Thali.
 
 ### License
-MIT
+
```

Comment:
We already have a dedicated license file, do we really want to repeat the license again?